src/main/java/camp/nextstep/edu/calculator/Number.java

```diff
@@ -0,0 +1,41 @@
+package camp.nextstep.edu.calculator;
+
+import java.util.Objects;
+
+public final class Number {
+    public static final int ZERO_VALUE = 0;
+    private int value;
+
+    private Number(final int number) {
+        if (isNegative(number)) {
+            throw new RuntimeException(String.format(Constants.RUNTIME_EXCEPTION_MESSAGE, number));
+        }
+        this.value += number;
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        this.value = number;
```